unison-src/transcripts/alias-many.output.md

```diff
@@ -455,11 +455,11 @@ I want to incorporate a few more from another namespace:
 .runar> find
 
   1.  List.adjacentPairs : [a] -> [(a, a)]
-  2.  List.all : (a ->{g} Boolean) ->{g} [a] ->{g} Boolean
-  3.  List.any : (a ->{g} Boolean) ->{g} [a] ->{g} Boolean
+  2.  List.all : (a ->{g} Boolean) -> [a] ->{g} Boolean
+  3.  List.any : (a ->{g} Boolean) -> [a] ->{g} Boolean
   4.  List.chunk : Nat -> [a] -> [[a]]
   5.  List.chunksOf : Nat -> [a] -> [[a]]
-  6.  List.dropWhile : (a ->{g} Boolean) ->{g} [a] ->{g} [a]
+  6.  List.dropWhile : (a ->{g} Boolean) -> [a] ->{g} [a]
```

Comment:
💯 

Nice that we got some extra ambient testing just from existing transcripts like this.